src/main/java/duke/exception/DukeException.java

```diff
@@ -0,0 +1,11 @@
+package duke.exception;
+
+public class DukeException extends Exception {
```

Comment:
DukeException should the base exception class for other, more specific exception types related to the application. If you get DukeException error, it gives you no more information about the type other than the message string.